src/components/Workout/create-workout-task.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from "react";
+import axios from "axios";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+export default class CreateWorkoutTask extends Component {
```

Comment:
why as class and not as function?

Reply:
User is written as a class so it might be more consistent to keep them as classes across the board